ops/cli.py

```diff
@@ -7,18 +7,18 @@
 usage: python3 cli.py -h
 """
 import argparse
+import getpass
```

Comment:
Nice, I've never heard of `getpass` before!

Reply:
Just a note that I'm going to revert the changed around cli.py (including this one), since the docker command does not work!